server/src/test/java/org/apache/iotdb/db/integration/IoTDBMultiSeriesIT.java

```diff
@@ -43,7 +44,7 @@
  */
 public class IoTDBMultiSeriesIT {
 
-  private static boolean testFlag = Constant.testFlag;
+  private static boolean testFlag = org.apache.iotdb.db.constant.TestConstant.testFlag;

```

Comment:
```suggestion
  private static boolean testFlag = TestConstant.testFlag;
```
And all the same appearance in this test class : )

Reply:
And also in other test classes...

Reply:
fixed